.github/workflows/tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             ${{ runner.os }}-
 
       - name: Install NPM dependencies
-        run: npm install purescript spago markdownlint-cli
```

Comment:
It doesn't work with newer versions

<img width="1058" alt="Screenshot 2024-07-20 at 13 28 49" src="https://github.com/user-attachments/assets/2e547dce-a7da-45af-a202-2ca2ebc51655">

Reply:
Very nice